packages/dds/tree/src/simple-tree/prepareForInsertion.ts

```diff
@@ -206,12 +207,12 @@ function walkMapTree(
 }
 
 /**
- * Register a collection of nodes with the forest so that they can be hydrated.
+ * Register events which will hydrate batches of nodes when they are inserted, assuming the next edits to forest are their insertions, in order.
```

Comment:
When we say "assuming the next edits to forest are their insertions" here, do we mean "register the events _if_ the subsequent edits are their insertions"? Or "register events, and the next edits _must_ be their insertions"?